shoreditch.php

```diff
@@ -158,4 +160,47 @@ function shoreditch_civicrm_pageRun(&$page) {
   if ($pageName == 'CRM_Contact_Page_View_Summary') {
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.shoreditch', 'js/contact-summary.js');
   }
+
+  _add_shoreditch_page_styles($page->urlPath);
+}
+
+/**
+ * @return bool
```

Comment:
Could we please provide a short description of what this func does? 

```
/**
 * Checks if custom Shoreditch CSS is enabled
 *
 * @return bool
 */
```

I know that this may sound a bit redundant hence the name of the function, but this is the standard we follow.